include/zephyr/bluetooth/mesh/dfu.h

```diff
@@ -35,6 +35,8 @@ extern "C" {
 #define CONFIG_BT_MESH_DFU_URI_MAXLEN 0
 #endif
 
+#define BT_MESH_BLOB_ID_UNDEFINED 0x00
```

Comment:
I'd rather not add it. It is only used for dfu_cli in `struct bt_mesh_dfu_cli_xfer` and description for `blob_id` parameter should be enough in my opinion.